src/xkbcomp/symbols.c

```diff
@@ -462,6 +463,12 @@ AddModMapEntry(SymbolsInfo *info, ModMapEntry *new)
         use = (clobber ? new->modifier : old->modifier);
         ignore = (clobber ? old->modifier : new->modifier);
 
+        // Remove entry
```

Comment:
This file only uses `/* */`-style comments

Reply:
Done